docs/running_tests.rst

```diff
@@ -62,3 +61,11 @@ server is found all elasticsearch tests will be skipped. Note that the tests
 are destructive - during the teardown phase they will wipe the cluster clean so
 make sure you don't run them against an instance with data you wish to keep.
 
+If you want to run the geo-django tests you'll probably need some additional
+settings, just run those commands with your own lib paths::
+
+	cd django-haystack/tests
+	export PYTHONPATH=`pwd`/..:`pwd`
+	export GEOS_LIBRARY_PATH=/opt/boxen/homebrew/lib/libgeos_c.dylib
```

Comment:
This isn't necessary for most installations and is covered by the Django documentation:

https://docs.djangoproject.com/en/1.7/ref/contrib/gis/install/geolibs/#geos-library-path

I think it makes the most sense to remove this or at most put a pointer to the Django documentation rather than either distracting people or having to maintain a parallel copy of the Django docs.